src/libs/ajax.js

```diff
@@ -411,6 +412,19 @@ export const DAR = {
     return dars;
   },
 
+  //new manage endpoint, should be renamed once v1 variant is removed from use
+  getDataAccessManageV2: async() => {
+    const url = `${await Config.getApiUrl()}/dar/manage/v2`;
+    const res = await axios.get(url, Config.authOpts());
+    return res.data;
+  },
+
+  updateDar: async (dar, id) => {
+    const url = `${await Config.getApiUrl()}/dar/${id}`;
+    const res = await fetchOk(url, fp.mergeAll([Config.authOpts(), Config.jsonBody(dar), { method: 'PUT' }]));
+    return await res;
+  },
```

Comment:
This was removed in https://github.com/DataBiosphere/duos-ui/pull/799. I don't see this used in your code, so it looks like you might need to rebase.